packages/@aws-cdk/aws-ec2/lib/bastion-host.ts

```diff
@@ -0,0 +1,76 @@
+import { PolicyStatement } from "@aws-cdk/aws-iam";
+import { CfnOutput, Construct } from "@aws-cdk/core";
+import { AmazonLinuxGeneration, AmazonLinuxImage, InstanceClass, InstanceSize, InstanceType } from ".";
+import { Instance } from "./instance";
+import { Port } from "./port";
+import { IVpc, SubnetType } from "./vpc";
+
+/**
+ * Properties of the bastion host
+ */
+export interface BastionHostProps {
+
+    /**
+     * In which AZ to place the instance within the VPC
+     *
+     * @default - Random zone.
+     */
+    readonly availabilityZone?: string;
+
+    /**
+     * VPC to launch the instance in.
+     */
+    readonly vpc: IVpc;
+
+    /**
+     * The name of the instance
+     *
+     * @default - CDK generated name
```

Comment:
Seems like the default is `"BastionHost"`

Reply:
The default is the node name. So `X/Y/BastionHost`